src/register-commands.ts

```diff
@@ -37,6 +39,11 @@ const offsetFlagConfig = {
     description: "specify a positive number of days in the past"
 };
 
+const allFlagConfig = {
+    flags: "--all",
+    description: "finish all of a days timers"
```

Comment:
Grammar pedantry: "finish all of a day's timers"

Reply:
Fixed